Solutions/Digital Shadows/Playbooks/readme.md

```diff
@@ -0,0 +1,35 @@
+# DigitalShadowsPlaybook-UpdateIncidentStatus
+
+## Overview
+The DigitalShadowsPlaybook-UpdateIncidentStatus is triggered when an alert is created by the Digital Shadows Data Connector Analytic rules. The main purpose of the playbook is to updates the Azure Sentinel Incident Status as per the logs received by Data connector. Following is the sequence of events take place for triggering the playbook:
+
+1. Digital Shadows Data Connector fetches the Triage Items from Digital Shadows Searchlight portal
+1. The analytic rule identifies an update in the Triage Item as an alert
+1. Each alert with new triage item (indentified by the unique triage item id) generates an Azure Sentinel Incident
+1. All the updates detected by Analytic rule are identified as alerts and correlated with an incident if the Incident (with same triage item id) already exists
+1. Creation of each alert triggers the "DigitalShadowsPlaybook-UpdateIncidentStatus" playbook so that the "Status" of the incident, associated with alert's triage item id, is updated
+
+## Playbook Workflow
+Following is the workflow of the playbook:
+1. Playbook is triggered on alert creation
+1. Get incident associated with the alert
+1. Get the custom details from the alert
+1. Retrieve the updated value for Status
+1. Update the Incident Status by mapping the Digital Shadows Status to corresponding Azure Sentinel Incident Status
+
+## Prerequisite
+1. Prior to the deployment of this playbook, Digital Shadows Data Connector needs to be deployed under the same subscription.
+1. Obtain Digital Shadows SearchLight API credentials from Digital Shadows. These credentials will be needed to configure the connector.
+
+## Deploying the Playbook
+The Playbook is deployed with the Digital Shadows Data Connector. To deploy the Data connector, follow the below steps:
+1. Go to Azure Sentinel -> Data Connectors
+1. Click on the Digital Shadows connector, connector page will open. 
+1. Click on the `Deploy to Azure` button.   
```

Comment:
As in the example it would be useful to have manual deployment instructions that include the 'Deploy to Azure' buttons shown in the example given by @devikamehra .

When deploying manually, I have also previously needed to complete further steps. For instance, after deploying the workbook I have had to update the IAM permissions of the workspace to give the 'Sentinel Responder' role to the playbook so that it could update the incident state. I have also then had to update the analytic rules to add this playbook as an automated response. Are those steps necessary when deploying the Data Connector? I suspect they are since the ARM deployment template for the data connector doesn't include this playbook.

Reply:
Adding to what @ds-jdcockrill mentioned, data connector is not required to be part of playbook pre-requisite. This readme should focus on the steps which are required for the standalone deployment of this playbook.

It is understood that with solution deployment, all the connected content would get installed. Please following the example to learn more about the README.

Reply:
@ds-jdcockrill I never had to update IAM permissions iirc, although we do have to add it in the analytic rule so that it triggers whenever the rule triggers. @devikamehra confirms in above comment that all of it is contained in a solution so will be installed together.

@devikamehra , this playbook is not intended to be a standalone deployment and without data connector log data it wont work as intended. If we have to have it as a standalone playbook then I will add the button and remove the data connector as a requirement.

Reply:
Hi @devikamehra - your comments should now have been addressed fully - could you please re-review and confirm your acceptance?

Thanks

Reply:
All the required changes are included here.